resources/js/components/topNavigationBar.vue

```diff
@@ -0,0 +1,156 @@
+<template >
```

Comment:
no need of vue in this, you can do the same thing in blade. Only use vue when you need reactivity.